test/ForwardDiff_long_test.jl

```diff
@@ -47,7 +46,7 @@ function test_vs_fin_diff(fun, p; ε=√eps(eltype(p)), kwargs...)
     isapprox(FD_grad, fin_diff_grad; kwargs...) ? true : (@show abs_diff, rel_diff; false)
 end
 
-@testset "ForwardDiff with schroedinger" begin
+@testitem "ForwardDiff with schroedinger" begin
```

Comment:
as long as this stays as a testset it will not be executed.

or even better, mark it as "tags = [:slow]" and add a filter that excludes slow tests

Reply:
Oh, should we also include this test file in the testing?

Reply:
TestItems automatically detects all `@testitem` in the codebase, no need for any explicit inclusions. This particular file was never executed because it is a very slow test -- we just kept it in case of debugging needs, etc. With testitems this can be done more neatly -- mark it as `:slow` and filter it out.